src/games/supported/BreakoutSuper.hpp

```diff
@@ -0,0 +1,87 @@
+/* *****************************************************************************
+ * The lines 68 and 76 are based on Xitari's code, from Google Inc.
```

Comment:
I just copied and pasted code from the Breakout.cpp file. I did not know to which extend that line was necessary.